custom/setup.py

```diff
@@ -0,0 +1,53 @@
+import os
+import itertools
+
+from typing import Dict, List
+from setuptools import setup, find_packages
+
+ROOT_PATH = os.path.dirname(__file__)
+PKG_NAME = "mlservermllib"
+PKG_PATH = os.path.join(ROOT_PATH, PKG_NAME)
+
+
+def _load_version() -> str:
+    version = ""
+    version_path = os.path.join(PKG_PATH, "version.py")
+    with open(version_path) as fp:
+        version_module: Dict[str, str] = {}
+        exec(fp.read(), version_module)
+        version = version_module["__version__"]
+
+    return version
+
+
+def _extras() -> Dict[str, List[str]]:
+    extras = {
+        "mllib": ["pyspark==3.0.1"]
+    }
+
+    # Inject key 'all' with all dependencies
+    all_extras = set(itertools.chain(*extras.values()))
+    extras.update({"all": list(all_extras)})
+
+    return extras
+
+
+setup(
+    name=PKG_NAME,
+    version=_load_version(),
+    url=f"https://github.com/seldonio/{PKG_NAME}.git",
+    author="Seldon Technologies Ltd.",
+    author_email="hello@seldon.io",
+    description="ML server",
+    packages=find_packages(),
+    install_requires=[
+        "grpcio==1.32.0",
```

Comment:
I can see that these `requires` are the same as the top-level ones from the `mlserver` package. Would it make sense to replace them and simply have the following?

```
install_requires=[
	mlserver,
	pyspark
]
```

Reply:
Put the `mlserver` in the requires means we don't need to `pip install mlserver mlserver-mllib` but just `pip install mlserver-mllib`.

Reply:
True. I think that's fair.